utils/benches/benches.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2023 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use criterion::{criterion_group, criterion_main, Criterion};
+
+use crypto::random::make_pseudo_rng;
+use utils::bloom_filters::rolling_bloom_filter::RollingBloomFilter;
+
+pub fn rolling_bloom_bench(c: &mut Criterion) {
+    let mut data = [0u8; 32];
+    let mut rng = make_pseudo_rng();
+
+    let mut list = RollingBloomFilter::<[u8; 32]>::new(120000, 0.000001, &mut rng);
+    let mut count = 0u32;
+    c.bench_function("RollingBloomFilter", |b| {
+        b.iter(|| {
+            count += 1;
+
+            data[0..4].copy_from_slice(&count.to_le_bytes());
+            list.insert(&data, &mut rng);
+
+            data[0..4].copy_from_slice(&count.to_be_bytes());
+            list.contains(&data);
```

Comment:
```suggestion
            list.contains(&data)
```
I doubt it will affect the results, but the return value is passed to `black_box`.

Reply:
Nice catch, thanks. I tried with and without it, and fortunately there is no difference (otherwise all my previous benchmarks would not be accurate). I added an explicit call to `black_box` which should be more noticeable.